src/plugins/recordTypes/dutyofcare/columns.js

```diff
@@ -0,0 +1,17 @@
+import { defineMessages } from 'react-intl';
+
+export default () => ({
+  default: {
+    dutyOfCareNumber: {
+      messages: defineMessages({
+        label: {
+          id: 'column.dutyofcares_common.default.dutyOfCareNumber',
+          defaultMessage: 'Duty of care id',
+        },
+      }),
+      order: 10,
+      sortBy: 'dutyofcares_common:referenceNumber',
+      width: 200,
+    },
```

Comment:
Looks like this is also missing `updatedAt` and the `narrow` definition. Going to get them in as well.

Reply:
Although looking at the other procedures I see they don't include `narrow` (although I thought some did). It doesn't seem like it will ever be used since it's only the related record sidebar that uses the `narrow` column type.

Reply:
You only need to define `narrow` if it's different than `default`. The sidebar will fall back if `narrow` doesn't exist. It's useful if you want to have fewer columns, or shorten the column labels to fit.